portal-api/src/v1/api.js

```diff
@@ -267,14 +267,33 @@ const saveUtilisationReport = async (reportData, reportPeriod, user, fileInfo) =
   }
 };
 
+/**
+ * Gets utilisation reports for a specific bank. If a report
+ * period is not provided, all reports for that bank are
+ * returned. If a report period is provided, the report
+ * submitted for that report period is returned. Returned
+ * reports are ordered by year descending and month
+ * ascending.
```

Comment:
Looks like both year and month are ascending?
```
return sortBy(filteredUtilisationReports, ['reportPeriod.start.year', 'reportPeriod.start.month']);
```

Reply:
I was getting confused with how something else worked (or how it used to work)